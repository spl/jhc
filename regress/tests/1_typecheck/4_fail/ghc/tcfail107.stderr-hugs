ERROR "tcfail107.hs":13 - Not enough arguments for type synonym "Const"
