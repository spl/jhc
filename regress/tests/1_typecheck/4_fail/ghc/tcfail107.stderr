
tcfail107.hs:13:1:
    Type synonym `Const' should have 2 arguments, but has been given 1
    In the type signature for `test':
      test :: Thing (Const Int) -> Thing (Const Int)
